reference/commands/graph/info.rst

```diff
@@ -130,5 +130,379 @@ The ``conan graph info`` command shows information about the dependency graph fo
 
 .. warning::
 
-  The json output of the ``conan graph --format=json`` is **experimental** and subject to
+  The json output of the ``conan graph info --format=json`` is **experimental** and subject to
   change.
+
+
+**Examples**:
+
+.. code-block:: bash
+
+    $ conan info .
+    $ conan info myproject_folder
+    $ conan info myproject_folder/conanfile.py
+    $ conan info hello/1.0@user/channel
```

Comment:
This is outdated:

- command is ``conan graph info``
- only ``conan graph info --requires=xxxx`` is supported.

Reply:
Copy and paste error oops 🤦